packages/desktop-client/src/components/accounts/Account.tsx

```
@@ -911,6 +923,35 @@
     return amount;
   };
 
+  getSelectedAccounts = (): AccountEntity[] => {
+    switch (this.props.accountId) {
+      case 'onbudget':
+        return this.props.accounts.filter(account => !account.offbudget);
+        break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)